draft-ietf-quic-manageability.md

```diff
@@ -863,11 +907,167 @@ security-relevant.
 
 Dan Druta contributed text to {{sec-ddos-dec}}. Igor Lubashev contributed text
 to {{sec-loadbalancing}} on the use of the connection ID for load balancing.
-Marcus Ilhar contributed text to {{sec-rtt}} on the use of the spin bit.
+Marcus Ilhar contributed text to {{sec-rtt}} on the use of the spin bit. The 
+pseudo provided in the appendix is based on input provided by David Schinazi.
 
 # Acknowledgments
 
+Thanks to Martin Thomas and Martin Duke for contributing by reviewing and 
+providing text proposals.
+
 This work is partially supported by the European Commission under Horizon 2020
 grant agreement no. 688421 Measurement and Architecture for a Middleboxed
 Internet (MAMI), and by the Swiss State Secretariat for Education, Research, and
 Innovation under contract no. 15.0268. This support does not imply endorsement.
+
+# Appendix
+
+## Handling of Google QUIC  {#sec-google-version} 
```

Comment:
I agree with @martinthomson here: beyond this section (mentioning the version numbers) and its first subsection (explaining how to differentiate Google and IETF QUIC), I'd scrub references to Google QUIC from the rest of the pseudocode, and split the appendix in two (first appendix: recognizing google quic; second appendix: initial salt (IETF only) and crypto frame extraction (IETF only)

Reply:
Okay, I removed the salt section entirely (because I think that value should be in the tls draft and only there) and cut down the crypto frame part to IETF QUIC only.